### PR TITLE
Track voluneteers add buttons

### PR DESCRIPTION
**Pull request by Ebenezer997**
Issue:  the search bar is visible when there are only a few data

Fix: Hid the search bar of the data tables when the data rows are few(less than or equal to 10). the pagination of the page has to be hidden when data table is only one page

### CODE REVIEW

**Comment by bryantal** [comment]
You'll need to merge this branch with development

**Comment by BrianRamsay** [comment]
Ebenezer, make sure your email address in your commits matches your email address in GitHub. Right now it is saying the commits are from "None"

File: app/static/js/trackVolunteers.js [hunk 1]
```diff
@@ -1,5 +1,43 @@
 import searchUser from './searchUser.js'
 
+$(document).ready( function () {
+   var table =  $('#trackVolunteerstable').DataTable({
+   });
+   console.log(table)
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: don't leave debug output

File: app/static/js/trackVolunteers.js [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+      //     "searching": false,;
+      // },
+      // responsive: true
+       // columnDefs: [
+       //  { responsivePriority: 6 ,targets: 6},
+       //  { responsivePriority: 5 ,targets: 5},
+       //  { responsivePriority: 4 ,targets: 4},
+       //  { responsivePriority: 3 ,targets: 3},
+       //  { responsivePriority: 2 ,targets: 2},
+       //  { responsivePriority: 1 ,targets: 1}
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: don't leave unused commented out code

File: app/templates/events/trackVolunteers.html [hunk 3]
```diff
@@ -3,10 +3,15 @@
 {% block scripts %}
   {{super()}}
   <script type="module" src="/static/js/trackVolunteers.js"></script>
+  <!-- <script type="module" src="//cdn.datatables.net/1.11.3/js/jquery.dataTables.min.js"></script> -->
+  <script type="text/javascript" src="https://cdn.datatables.net/v/dt/dt-1.11.3/r-2.2.9/datatables.min.js"></script>
```
Comment (BrianRamsay) [inline app/templates/events/trackVolunteers.html:6]: This seems very specific. Do we need to use such a specific version of datatables or can we just use the latest for our jquery version

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/static/js/trackVolunteers.js [hunk 4]
```diff
@@ -1,5 +1,43 @@
 import searchUser from './searchUser.js'
 
+$(document).ready( function () {
+   var table =  $('#trackVolunteerstable').DataTable({
+   });
```
Comment (BrianRamsay) [inline app/static/js/trackVolunteers.js]: This shouldn't be on its own line

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/templates/events/trackVolunteers.html (hunk 3 above)
Comment (Ebenezer997) [reply app/templates/events/trackVolunteers.html:6]: yes, we keep it. This version is for the responsiveness of the data tables.

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**